public/app/core/components/Layers/LayerDragDropList.tsx

```diff
@@ -122,10 +122,6 @@ export const LayerDragDropList = <T extends LayerElement>({
   );
 };
 
-LayerDragDropList.defaultProps = {
-  isGroup: () => false,
-};
-
```

Comment:
sure - the reason i just removed it is because it's not listed in the prop interface or passed as a prop anywhere, so i'm not sure it was actually doing anything before 😅